packages/contracts-bedrock/src/L1/L1CrossDomainMessenger.sol

```diff
@@ -69,6 +70,50 @@ contract L1CrossDomainMessenger is CrossDomainMessenger, ISemver {
         return portal;
     }
 
+    /// @notice send the cross domain message for minting ETH ERC20 tokens on the L2.
+    /// @param _l1Token The address of the L1 token.
+    /// @param _l2Token The address of the L2 token.
+    /// @param _sender The address of the sender.
+    /// @param _to The address of the recipient.
+    /// @param _value The amount of tokens to mint.
+    /// @param _gasLimit The gas limit for the message.
+    /// @return The cross domain message.
+    function sendMintETHERC20Message(
+        address _l1Token,
+        address _l2Token,
+        address _sender,
+        address _to,
+        uint256 _value,
+        uint64 _gasLimit
```

Comment:
No verification were made to the addresses nor the value/gasLimit

Reply:
This function is triggered in OptimismPortal and we control the inputs, so there is no need to verify them.